river/main.zig

```diff
@@ -41,7 +41,16 @@ const usage: []const u8 =
 
 pub var server: Server = undefined;
 
+fn sa_handler(_: c_int) callconv(.C) void {}
+
 pub fn main() anyerror!void {
+    const act = os.Sigaction{
+        .handler = .{ .handler = sa_handler },
+        .mask = os.empty_sigset,
+        .flags = 0,
+    };
+    os.sigaction(os.SIG.PIPE, &act, null);
+
```

Comment:
It would be nice to have a short comment in the source code to explain why we ignore SIGPIPE and why we use an empty handler instead of `SIG_IGN`. Other than that this LGTM!